catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/schemas/camel-spring.xsd

```diff
@@ -9382,7 +9382,7 @@ The line separator of the files The default value is to use the JVM platform lin
           <xs:annotation>
             <xs:documentation xml:lang="en">
 <![CDATA[
-The normalized line separator of the files The default value is a new line character. Default value: \n
+The normalized line separator of the files The default value is a new line character. Default value: 
```

Comment:
Only here its not correct, but its something we can accept. Or you need to check the maven plugin that generates these comments: camel-eip-documentation-enricher-maven-plugin

Reply:
Yes, true; we could do this:  text += " Default value: " + defaultValueText.toString().replace("\n", "\\n");